apps/studio/electron/main/code/classes.ts

```diff
@@ -35,5 +35,23 @@ function getNodeClasses(node: t.JSXElement): string[] {
         return classNameAttr.value.expression.value.split(/\s+/).filter(Boolean);
     }
 
+    if (
+        t.isJSXExpressionContainer(classNameAttr.value) &&
+        t.isTemplateLiteral(classNameAttr.value.expression)
+    ) {
+        const templateLiteral = classNameAttr.value.expression;
+
+        // Checks if all classes in the expression are static
+        const allStatic = templateLiteral.expressions.length === 0;
+
+        // Returns the classes if all are static, otherwise returns a message
+        if (allStatic) {
+            const quasis = templateLiteral.quasis.map((quasi) => quasi.value.raw.split(/\s+/));
+            return quasis.flat().filter(Boolean);
+        } else {
+            return ['Dynamic classes detected.'];
```

Comment:
Could we return a special type instead? Maybe an error type with a flag and message? 
This way we can use special logic to handle the type. Something like this with better naming? 

My thinking eventually we may want to return objects to be used. This way we can extend the return types and add custom logic.

```typescript

interface ClassesReturnValue {
type: 'classes',
value: string[]
}

interface ErrorReturnValue {
type: 'error',
reason: string
}

type ReturnValue = ClassesReturnValue | ErrorReturnValue
```

What do you think @cleobnvntra ?

Reply:
Yeah I think that's a good idea. It is easily customizable and extensible with an interface rather than just returning a string. Easily doable too as it seems like the functions involved here is not used anywhere else other than in `TailwindInput/index.tsx`.

Can add the implementation to the PR later after double checking for return type conflicts, and a better interface name 😆